createAccountCore/src/test/java/com/hamdan/forzenbook/createaccountcore/domain/DomainUnitTest.kt

```diff
@@ -0,0 +1,100 @@
+package com.hamdan.forzenbook.createaccountcore.domain
```

Comment:
The bin and txt files you have in the changed file list under this file need to not be checked in

Reply:
removed them